security-policy.md

```diff
@@ -4,4 +4,166 @@ title: Security Policy
 breadcrumbs: true
 ---
 
-TODO: Put Policy
\ No newline at end of file
+## Introduction
+
+This document outlines the security policy for our CVE Numbering Authority
+(CNA). It details our scope, contact methods, disclosure processes, report
+formats, dispute handling procedures, and references to our Root CNA's policies.
+
+## 1. Scope
+
+Our detailed scope is defined in our [Scope Definition](/scope).
+
+## 2. Contact Methods
+
+We encourage security researchers and members of the community to report
+vulnerabilities through the contact methods outlined on our
+[Contact Page](/contact).
```

Comment:
We should add the `security.txt`, if it is not yet in EEF's website.
This file is mandatory by some gov agencies.
Random link from Swiss National Cyber Security Center: https://www.ncsc.admin.ch/securitytxt-en

Reply:
this may need not be mentioned here, I don't know...

Reply:
@kikofernandez I don’t think we need to mention it here. But adding it is a good idea 👍

Reply:
@kikofernandez See #14